### ISSUE TITLE
Question about pit loss calculation in EDA-EEND model

### ISSUE THREAD

**Issue by SoundingSilence** [hitachi-speech/EEND#44]
Thanks for your valuable work on EEND. In eend/chainer_backend/models.py (line 491-497), why the loss is calculated twice, substituted with standard loss finally? 

    loss, labels = batch_pit_n_speaker_loss(ys_padded, ts_padded, n_speakers)

    loss = standard_loss(ys, labels)

I am looking forward to your reply. Thanks!
